net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -322,32 +322,37 @@ mwan3_create_iface_route()
 	[ -n "$id" ] || return 0
 
 	if [ "$family" == "ipv4" ]; then
-		ubus call network.interface.${1}_4 status &>/dev/null
-		if [ "$?" -eq "0" ]; then
```

Comment:
Could you do this please in a separate commit?
This is not reflected in the commit message.
And has nothing to do with this commit.

Reply:
It simply does exactly the same thing as the origin code and it reduces one extra comparison.
I've seen many of such superfluous usage in the code. If you don't like I can just undo it. :(